types/src/space.d.ts

```diff
@@ -27,15 +38,34 @@ export interface CoordMeta {
 }
 
 export interface Options {
+	/** Id of this space, used in things such as conversions */
 	id: string;
+	/** The readable name of the space, used in user-facing UI */
 	name: string;
+	/** The base color space */
 	base?: string | ColorSpace | null | undefined;
+	/**
+	 * Function that converts coords in the base color space to coords in this color space.
+	 * Must perform chromatic adaptation if needed
+	*/
 	fromBase?: ((coords: Coords) => number[]) | undefined;
+	/**
+	 * Function that converts coords in this color space to coords in the base color space.
+	 * Must perform chromatic adaptation if needed
+	 */
 	toBase?: ((coords: Coords) => number[]) | undefined;
+	/**
+	 * Object mapping coord ids to coord metadata
+	 * @see {@link CoordMeta}
+	*/
 	coords?: Record<string, CoordMeta> | undefined;
 	white?: string | White | undefined;
 	cssId?: string | undefined;
```

Comment:
Probably worth documenting this, as the id used by CSS, e.g. `display-p3` or `--cam16-jmh`.